tc/sgx/trusted_worker_manager/enclave/work_order_processor.cpp

```diff
@@ -444,6 +444,8 @@ namespace tcf {
 
         JsonSetNumber(error, "code", err_code, "failed to serialize error code");
         JsonSetStr(error, "message", err_message, "failed to serialize error message");
+        JsonSetStr(error, "workOrderId", work_order_id.c_str(),
```

Comment:
workOrderId should be inside data field as below
"data": {
"workOrderId": <work_order_d>
}

Reply:
Ok, will make the correction